src/toolchain/toolchain.ts

```diff
@@ -205,6 +210,27 @@ export class SwiftToolchain {
         return undefined;
     }
 
+    /**
+     * @param target Target to obtain the SDK path for
+     * @returns path to the SDK for the target
+     */
+    public static async getSdkForTarget(
+        target: DarwinCompatibleTarget
+    ): Promise<string | undefined> {
+        let sdkType: string;
+        switch (target) {
+            case DarwinCompatibleTarget.macOS:
+                sdkType = "macosx";
+                break;
+            case DarwinCompatibleTarget.iOS:
+                sdkType = "iphoneos";
+                break;
+        }
+
+        const { stdout } = await execFile("xcrun", ["--sdk", sdkType, "--show-sdk-path"]);
```

Comment:
Can you include the following as an argument in the `execFile` call. Means we can pass `DEVELOPER_DIR` to the `xcrun` call and get the sdk from non-default Xcode installs 
```
options: {
    env: {...process.env, ...configuration.swiftEnvironmentVariables},
}
```

Reply:
Sure, no problem. That seems reasonable to me. I've added the environment variable forwarding to my changes.